### PR TITLE
Update to 2.7.4_1

### PR DESCRIPTION
**Pull request by dennypage**

### CODE REVIEW

File: sysutils/pfSense-pkg-nut/pkg-descr [hunk 1]
```diff
@@ -1 +1,3 @@
-Network UPS Tools
+Network UPS Tools provides support for monitoring of Uninterruptible Power Supplies. It supports UPS units attached locally via USB or serial, and remote units via the SNMP protocol, the APCUPSD protocol or the NUT protocol.
```
Comment (rbgarga) [inline sysutils/pfSense-pkg-nut/pkg-descr]: pkg-descr must respect 80 columns size. Could you please wrap this long line?

**Comment by dennypage** [comment]
Sorry, I didn't know of the limit. Change done.

**Comment by rbgarga** [comment]
yeah, it's a FreeBSD ports rule. If you want to validate a port you can use portlint - https://www.freebsd.org/doc/en/books/porters-handbook/testing-portlint.html

**Comment by dennypage** [comment]
Thank you Renato, much appreciated.

**Comment by dennypage** [comment]
I added some changes to the Makefile requested by portlint. Thanks again!

**Comment by dennypage** [comment]
Hey Renato, I've updated for 2.7.4_1. Please let me know if there is anything else you see that needs fixing. Thanks!

**Pull request merged**